neuralhydrology/datasetzoo/camelsind.py

```diff
@@ -0,0 +1,345 @@
+
+    cols_to_drop = ['gauge_id', 'year', 'month', 'day']
+    for basin_id, basin_df in tqdm(merged_df.groupby('gauge_id'), desc="Writing time series files"):
+        basin_df = basin_df.copy()
+        basin_df.set_index('date', inplace=True)
+        basin_df.drop(columns=cols_to_drop, inplace=True)
+        basin_df.to_csv(timeseries_output_dir / f"{basin_id}.csv")
+        
+    # --- NEW: Print a summary of the available dynamic features ---
+    print("\n--- Summary of Available Dynamic Features ---")
+    # Define columns that are for utility/indexing, not features
+    utility_columns = ['gauge_id', 'year', 'month', 'day', 'date']
+    # Get the dynamic feature names by filtering out utility columns
+    dynamic_features = [col for col in merged_df.columns if col not in utility_columns]
+    
+    # Separate into likely inputs and targets for clarity
+    potential_targets = ['streamflow']
+    dynamic_inputs = [feat for feat in dynamic_features if feat not in potential_targets]
+    targets = [feat for feat in dynamic_features if feat in potential_targets]
+    
+    print("The following columns are now available in each per-basin CSV file.")
+    print("Use these names in your NeuralHydrology config.yml file:")
+    print(f"  > For 'dynamic_inputs': {dynamic_inputs}")
+    if targets:
+        print(f"  > For 'target_variables': {targets}")
+    else:
+        print("  > Warning: No standard target variable like 'streamflow' was found.")
+
+    print(f"\nAll per-basin time series files have been stored in: {timeseries_output_dir}")
+
+
+
+def _process_and_merge_attributes(base_path: Path, attributes_output_file: Path):
+    """
+    Handles the merging of all static attribute files into a single CSV.
+    """
+    print("\n--- Step 2: Merging All Attribute Files ---")
+    attributes_folder = base_path / 'attributes_csv'
+    
+    if not attributes_folder.is_dir():
+        raise FileNotFoundError(f"Attributes folder not found at: {attributes_folder}")
+
+    csv_files = list(attributes_folder.glob('*.csv'))
+    if not csv_files:
+        print("Warning: No attribute CSV files found. Skipping attribute processing.")
+        return
+
+    dataframes = [pd.read_csv(file) for file in tqdm(csv_files, desc="Reading attribute files")]
+    
+    merged_attributes_df = reduce(
+        lambda left, right: pd.merge(left, right, on='gauge_id', how='outer'), 
+        dataframes
+    )
+    
+    merged_attributes_df = merged_attributes_df.dropna(axis=1, how='all')
+    merged_attributes_df.to_csv(attributes_output_file, index=False)
+    print(f"Successfully merged {len(csv_files)} attribute files.")
+    print(f"Consolidated attributes file saved to: {attributes_output_file}")
+
+
+# --- DOCSTRING UPDATED ---
+def preprocess_camels_ind_dataset(camels_base_path: str, output_dir: str):
+    """
+    Orchestrates the full preprocessing of a CAMELS-IND dataset. The dataset can be downloaded from
+    <https://zenodo.org/records/14999580>
```

Comment:
Missing one line doc string. Also I think the link will not correctly render in the sphinx docs. Maybe use the way you also reference the dataset in the doc string of the class?